app/features/__init__.py

```diff
@@ -9,3 +10,6 @@ class FeaturesContainer(containers.DeclarativeContainer):
     content = providers.Container(
         ContentContainer, resources=resources, config=config.content
     )
+    bot = providers.Container(
```

Comment:
should be twitch container -> and inside twitch container you can make bot, and channel containers for query